CleansePlayground.playground/Pages/GithubClientExample.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,126 @@
+/*:
+ 
+ ## Github Client Example.
+
+ This is a demonstration of using Cleanse and NSURLSession and friends to configure a github client that
+ will list a User's projects
+*/
+
+import Cleanse
+
+import XCPlayground
+
+
+//: Since we will be doing an asynchronous call, we need to make it so the execution doesn't stop immediately
+XCPlaygroundPage.currentPage.needsIndefiniteExecution = true
+
+//: This is the base URL for github
+struct GithubBaseURL : Tag {
+    typealias Element = NSURL
+}
+
+//: The `NetworkModule` makes an `NSURLSession` available
+struct NetworkModule : Module {
+    func configure<B : Binder>(binder binder: B) {
+
+//: Make `NSURLSessionConfiguration.ephemeralSessionConfiguration` be provided when one requests a `NSURLSessionConfiguration`
+        binder
+            .bind()
+            .asSingleton()
+            .to(factory: NSURLSessionConfiguration.ephemeralSessionConfiguration)
+
+//: Make `NSURLSession` available. It depends on `NSURLSessionConfiguration` configured above (`$0`)
+        binder
+            .bind()
+            .asSingleton()
+            .to {
+                NSURLSession(
+                    configuration: $0,
+                    delegate: nil,
+                    delegateQueue: NSOperationQueue.mainQueue()
+                )
+            }
+    }
+}
+
+//: Simple Service that lists "Member" logins for the current organization
+protocol GithubListMembersService {
+    func listMembers(organizationName: String, handler: [String] -> ())
```

Comment:
style nit (feel free to ignore): I like to name the parameters to my handlers for easier legibility